.github/workflows/build-and-test-studio.yaml

```diff
@@ -151,3 +121,47 @@ jobs:
         with:
           name: cypress-videos
           path: cypress/videos
+
+  publish:
+    if: | 
+      github.repository == 'scilifelabdatacentre/stackn'
+    needs: build_and_test
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - dockerfile: ./Dockerfile
```

Comment:
Should we prefix the studio and ingress names with "serve-" similar to serve-images so that they can be easily identified in the packages list?
https://github.com/orgs/ScilifelabDataCentre/packages

Reply:
FIXED!